lib/postprocessing/image.js

```diff
@@ -302,11 +337,13 @@ async function write(img, outfile) {
 
 // exported function
 async function imagePostProcess(rendition, outfile) {
+    const outdir = path.dirname(outfile);
```

Comment:
Actually a "working directory" should be passed in by the worker, who manages the directory structures here. (It can still be the same directory where the rendition (outfile) is created, but it is possible we want to change this in the future, and thus the decision does not belong in here).

Would name it `workDir` instead of `outdir` then.